src/app/clusters/microwave-oven-control-server/microwave-oven-control-server.cpp

```diff
@@ -300,20 +300,20 @@ bool IsPowerSettingInRange(uint8_t powerSetting, uint8_t minCookPower, uint8_t m
 
 void SetOPInstance(EndpointId aEndpoint, OperationalState::Instance * aInstance)
 {
-    uint16_t ep = emberAfGetClusterServerEndpointIndex(aEndpoint, MicrowaveOvenControl::Id,
-                                                       EMBER_AF_MICROWAVE_OVEN_CONTROL_CLUSTER_SERVER_ENDPOINT_COUNT);
+    uint16_t aMirowaveOvenEpIndex = emberAfGetClusterServerEndpointIndex(
+        aEndpoint, MicrowaveOvenControl::Id, EMBER_AF_MICROWAVE_OVEN_CONTROL_CLUSTER_SERVER_ENDPOINT_COUNT);
     // if endpoint is found
-    if (ep < kMicrowaveOvenControlInstanceTableSize)
+    if (aMirowaveOvenEpIndex < kMicrowaveOvenControlInstanceTableSize)
     {
-        gOPInstanceTable[aEndpoint] = aInstance;
+        gOPInstanceTable[aMirowaveOvenEpIndex] = aInstance;
```

Comment:
Careful about typos. There is "aMirovaveOvenEpIndex" everywhere --> "aMicrowaveOvenEpIndex" is correct spelling.